CMakeLists.txt

```diff
@@ -688,21 +685,17 @@ if(NRN_ENABLE_PYTHON)
   message(STATUS "  LIB         | ${NRN_DEFAULT_PYTHON_LIBRARIES}")
   message(STATUS "  MODULE      | ${NRN_ENABLE_MODULE_INSTALL}")
   message(STATUS "  DYNAMIC     | ${NRN_ENABLE_PYTHON_DYNAMIC}")
-  if(PYTHON_VERSION_MAJOR EQUAL 2)
-    set(PY2_WARNING TRUE)
-  elseif(PYTHON_VERSION_MINOR EQUAL 5)
-    set(PY35_WARNING TRUE)
+  if(PYTHON_VERSION_MAJOR LESS 3)
+    message(SEND_ERROR "Python 3 required. Please upgrade.")
   endif()
   if(NRN_ENABLE_PYTHON_DYNAMIC)
     list(LENGTH NRN_PYTHON_EXE_LIST _num_pythons)
     math(EXPR num_pythons "${_num_pythons} - 1")
     foreach(val RANGE ${num_pythons})
       list(GET NRN_PYTHON_EXE_LIST ${val} exe)
       list(GET NRN_PYTHON_VER_LIST ${val} version)
-      if(version EQUAL 2)
-        set(PY2_WARNING TRUE)
-      elseif(PYTHON_VERSION_MINOR EQUAL 5)
```

Comment:
Possibly pedantic comment: this drops the warning for 3.5 deprecation without actually removing 3.5 support.

Reply:
Well that was there for 8.0.0 release, not much sense in keeping any warning in `master` since I will do a PR for py3.5 removal just after.

Reply:
LGTM